src/main/java/org/codewars/kata/implementation/kovalchukvita/SixImpl.java

```diff
@@ -32,7 +32,7 @@ public String balance(String book) {
                     .append(String.format(" Balance %.2f", balance - sum)).append(book);
         }
 
-        return report + String.format("Total expense %.2f%sAverage expense %.2f", sum, book, sum / (str.length - 1));
+        return report + String.format("Total expense  %.2f%sAverage expense  %.2f", sum, book, sum / (str.length - 1));
```

Comment:
idk... mb woulg be better to separate requests?

Reply:
Liubomyr said that this pull request can be with change in this file